src/components/header/SubHeader.js

```diff
@@ -256,6 +264,19 @@ class Subheader extends Component {
                         {keymap.GLOBAL_CONTEXT.NEW_DOCUMENT}
                     </span>
                 </div>}
+                {gridView && query && (
+                    <a
+                        className="subheader-item js-subheader-item"
+                        href={`/rest/api/documentView/${windowType}/${query.viewId}/export/excel?selectedIds=${selected.join(',')}`}
```

Comment:
the URL is not OK. See
https://github.com/metasfresh/metasfresh-webui-frontend/blob/b55753626808dcac518c94d0972ab88cd0f9a940/config.js.dist#L2

Reply:
I took the URL schema from your [comment](https://github.com/metasfresh/metasfresh-webui-frontend/issues/1197#issue-257328355) - or do you refer to explicitly prepending `config.API_URL`? Maybe I'm overlooking something here.

Reply:
Oh, got it - I need to prepend `config.API_URL` and strip `/rest/api` from the hardcoded string.